### DISCUSSION TITLE
Apriori and FPGrowth returning short itemsets

### DISCUSSION THREAD

**Discussion by deleted-12** [rasbt/mlxtend#885] [Q&A] [answered]
Hello everyone, I hope you all doing well.

I gave both the apriori and fpgrowth functions a list of item sets that are from size 2 and larger, but the functions output itemsets start from size 1 not 2. Isn't this wrong for both algorithms?

**Comment by rasbt** [comment, marked as answer]
Itemsets can usually have any size from 1 to the number of possible items. You can set a higher support value to avoid the small itemsets.

**Reply by deleted-12** [reply]
Thank you
